draft-ietf-quic-recovery.md

```diff
@@ -435,7 +435,9 @@ Data in CRYPTO frames is critical to QUIC transport and crypto negotiation, so a
 more aggressive timeout is used to retransmit it.
 
 The initial crypto retransmission timeout SHOULD be set to twice the initial
-RTT.
+RTT.  On each consecutive expiration of the crypto timer without receiving an
+acknowledgement for a new packet, the sender SHOULD double the crypto
+retransmission timeout and set a timer for this period.
```

Comment:
Not quite right. If we got some acknowledgments, we still need to restart the timer. And note the pseudocode comments below.

Reply:
This was just a reorg.  This text is about doubling the timer after expiry when no ACKs are received.

I think the pseudocode comments are all addressed in the other PR.

Reply:
Ah, OK. Line 448 probably should read "When an acknowledgment is received, a new RTT is computed. The endpoint MUST restart the timer and SHOULD set it for twice the newly computed smoothed RTT."